packages/app/tsconfig.json

```diff
@@ -17,7 +17,9 @@
     "experimentalDecorators": true,
     "baseUrl": ".",
     "paths": {
-      "@/*": ["src/*"]
+      "@/*": ["src/*"],
+      "@pathfinder/data-services": ["../data-services/src/index"],
+      "@toeverything/pathfinder-logger": ["../logger/src/index"]
```

Comment:
there shouldn't be config by tsconfig, we depend on the package not the source code. because we have done some resolve modules when deploy to production environment.